src/Core/Protocol.php

```diff
@@ -182,7 +182,8 @@ public static function follow(int $uid, array $contact, ?string $protocol = null
 	public static function unfollow(array $contact, array $user): ?bool
 	{
 		if (empty($contact['network'])) {
-			throw new \InvalidArgumentException('Missing network key in contact array');
+			Logger::notice('Contact has got no network, we quit here', ['id' => $contact['id']]);
+			return true;
```

Comment:
Please return `null` instead, as no remote action was performed. A `null` return value will not defer the Unfollow task.